types/views/views.go

```diff
@@ -190,6 +210,22 @@ type Slice[T any] struct {
 	ж []T
 }
 
+// MapKey returns a unique key for a slice, based on its address and length.
+// It does not hash the contents of the slice.
```

Comment:
Remove the sentence ab out "hash the contents" since that's an implicit property of using `SliceMapKey` with a Go map.